astropy/coordinates/solar_system.py

```diff
@@ -374,7 +374,7 @@ def get_body_barycentric(body, time, ephemeris=None):
 get_body_barycentric.__doc__ += indent(_EPHEMERIS_NOTE)[4:]
 
 
-def _get_apparent_body_position(body, time, ephemeris):
+def _get_apparent_body_position(body, time, ephemeris, observer_loc):
```

Comment:
I'd just call it `obsgeoloc=None` as you do in the non-private function (and add a default; even though this is private, it may be used and we might as well not break such uses)